src/sentry/static/sentry/app/favicon.tsx

```diff
@@ -0,0 +1,25 @@
+function changeFavicon(theme: 'dark' | 'light'): void {
+  const n = document.querySelector<HTMLLinkElement>('[rel="icon"][type="image/png"]');
+  if (!n) {
+    return;
```

Comment:
here you return undefined, but you have a return type void... didn't you get any type warning?

Reply:
void here just means to never expect the function to return anything, its the same type that typescript would've inferred

Reply:
oh yes, you are right 😉